src/EventListener/ParamConverterListener.php

```diff
@@ -106,19 +106,12 @@ private function autoConfigure(\ReflectionFunctionAbstract $r, Request $request,
 
     private function getParamClassByType(?\ReflectionType $type): ?string
     {
-        if (\PHP_VERSION_ID < 80000) {
-            return (null === $type || $type->isBuiltin() || !$type instanceof \ReflectionNamedType) ?
-                null : $type->getName();
+        if (null === $type) {
+            return null;
         }
 
         foreach ($type instanceof \ReflectionUnionType ? $type->getTypes() : [$type] as $type) {
-            if (null === $type || $type->isBuiltin() || !$type instanceof \ReflectionNamedType) {
```

Comment:
💡 This is redundant. On PHP 7, `$type` is always `ReflectionNamedType` because that's the only implementation of `ReflectionType` there. On PHP 8.0, `ReflectionUnionType::getTypes()` always returns an array of `ReflectionNamedType` instances.